pyk/src/pyk/kcfg/explore.py

```diff
@@ -225,19 +222,19 @@ def extract_rule_labels(_logs: tuple[LogEntry, ...]) -> list[str]:
             log(f'abstraction node: {node_id}')
             return Abstract(abstract_cterm)
 
-        _branches = self.kcfg_semantics.extract_branches(_cterm)
-        branches = []
-        for constraint in _branches:
-            kast = mlAnd(list(_cterm.constraints) + [constraint])
-            kast, _ = self.cterm_symbolic.kast_simplify(kast)
-            if not CTerm._is_bottom(kast):
-                branches.append(constraint)
-        if len(branches) > 1:
-            constraint_strs = [self.pretty_print(bc) for bc in branches]
-            log(f'{len(branches)} branches using heuristics: {node_id} -> {constraint_strs}')
-            return Branch(branches, heuristic=True)
-
-        cterm, next_cterms, depth, vacuous, next_node_logs = self.cterm_symbolic.execute(
+        # _branches = self.kcfg_semantics.extract_branches(_cterm)
+        # branches = []
+        # for constraint in _branches:
+        #     kast = mlAnd(list(_cterm.constraints) + [constraint])
+        #     kast, _ = self.cterm_symbolic.kast_simplify(kast)
+        #     if not CTerm._is_bottom(kast):
+        #         branches.append(constraint)
+        # if len(branches) > 1:
+        #     constraint_strs = [self.pretty_print(bc) for bc in branches]
+        #     log(f'{len(branches)} branches using heuristics: {node_id} -> {constraint_strs}')
+        #     return Branch(branches, heuristic=True)
+
+        cterm, next_cterms_with_branch_constraints, depth, vacuous, next_node_logs = self.cterm_symbolic.execute(
```

Comment:
```suggestion
        cterm, next_states, depth, vacuous, next_node_logs = self.cterm_symbolic.execute(
```

(Makes more sense if you also define `NextState`.)

Reply:
Yes, done.